tests/test_scala.py

```diff
@@ -0,0 +1,29 @@
+def test_java_exists(Command):
```

Comment:
This file has a bunch of tabs in it, which I don't particularly care about, but `flake8` does. I think it would be good to adhere to PEP8 in these test files.